### ISSUE TITLE
Hyperparameters for the mobilenet model training

### ISSUE THREAD

**Issue by mrkaworu** [tensorlayer/hyperpose#163]
Hello everyone,

I've been able to successfully train both vgg-based models: vggtiny and vgg. Regretfully after trying with mobilenet I haven't been able to get the total loss below 300 mark. I did some testing using the AdamOptimizer from tensorflow but it didn't seem to have a big impact in the learning. Anyone has some tips about how to train the mobilenet version?

Cheers!

**Comment by zsdonghao** [comment]
Hello, 

The weight initialisation is very important for mobilenet model, and also the architecture has many tricks, for example, concating the low-level feature with the high-level features before stage 0 can give a better result, but will reduce the speed.

I didn't fully evaluate the mobilenet model actually, but I tried other peoples mobilenet implementations, the results are not well enough, especially for commercial application. Anyway, it is worth to train a better mobilenet with openpose-plus, as it provides industrial-level pipeline that can help to speed up the inferencing.

Feel free to discuss here.

Best,
